compiler/rustc_parse_format/src/lib.rs

```diff
@@ -819,12 +819,15 @@ fn find_skips_from_snippet(
     };
 
     fn find_skips(snippet: &str, is_raw: bool) -> Vec<usize> {
+        if is_raw {
```

Comment:
Can you pull this branch outside of the call to `find_skips`?
BTW, why do we need a `find_skips` function, can it be inlined into the caller?

Reply:
I fixed that.